Data-Structure/Collection-In-Java/CollectionOfEmployee/NaturalSorting.java

```diff
@@ -0,0 +1,24 @@
+package CollectionOfEmployee;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Niraj Gupta
+ * Class to perfrom natural sorting.
+ *
+ */
+public class NaturalSorting{

```

Comment:
the desired functionality could have also been achieved by making a method in same cls, do we have any specific reason or vision behind creating a new cls just fr one method

Reply:
The question in assignment asked to make three different program for the above three functions.

Reply:
ok